pages/k8s/release-notes.md

```diff
@@ -13,6 +13,61 @@ layout: [base, ubuntu-com]
 toc: False
 ---
 
+# 1.18
+
+### April xx, 2020 - [charmed-kubernetes-xxx](https://api.jujucharms.com/charmstore/v5/charmed-kubernetes-xxx/archive/bundle.yaml)
+
+Before upgrading, please read the [upgrade notes](/kubernetes/docs/upgrade-notes).
+
+## What's new
+
+- New SSL options for default ingress controller
+
+TODO: Write feature description based on https://bugs.launchpad.net/charm-kubernetes-worker/+bug/1859156 and https://github.com/charmed-kubernetes/charm-kubernetes-worker/pull/49/files
+
+- Multus support
+
+This release of Charmed Kubernetes introduces support for
+[Multus](https://github.com/intel/multus-cni), a CNI provider that makes it
+possible to attach multiple network interfaces to your pods.
+
+Along the way, we've also updated existing charms to make it possible for
+multiple CNI providers to be deployed together in the same cluster.
+
+Please note that while we are making Multus support available today, it is
+dependent on functionality in Juju that is not yet considered stable. For more
+details on the current state of Multus support in Charmed Kubernetes and how to
+get started, please refer to the
+[Multus documentation page](/kubernetes/docs/cni-multus).
```

Comment:
This link will be broken until https://github.com/charmed-kubernetes/kubernetes-docs/pull/367 is merged